source/overview/faq.rst

```diff
@@ -587,3 +587,21 @@ EXCEPT AS OTHERWISE EXPRESSLY AGREED BY THE PARTIES IN WRITING, MATTERMOST MAKES
 Does Mattermost answer questions about open source licenses authored by other organizations? 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 No, if you have questions about an open source license please consult the original author, or FAQs they offer. 
+
+What do I need to do to reset my password?
```

Comment:
I think this would be better in a different section than "Business". Maybe "Use Cases", "Product questions", or alternatively you could create a new section for this.

Reply:
Wondering if this should go in the end-user guide?

Reply:
@amyblais @wiersgallak Thanks for your input on this. The reason why I included this in the FAQ is due to the number of support tickets we have received related to password reset which (as we all know) should be handled by the System Administrator of their Mattermost instance. 

I do agree that I should have a new section for this nevertheless. Maybe create a new How To section? For example:

"How do I reset my password?"

Reply:
@ahmaddanialmohd How about "Account Questions"?

Reply:
I would expect to see some of this type of content in https://docs.mattermost.com/help/settings/account-settings.html#password or as a FAQ/Account Questions section generally on this page.

Reply:
> https://docs.mattermost.com/help/settings/account-settings.html#password

I agree with that and we could also add it to the Troubleshooting guide under the Administration section: https://docs.mattermost.com/install/troubleshooting.html#administration-issues as "Resetting user password"?